frontend/src/Components/Button/__test__/Button.test.tsx

```diff
@@ -0,0 +1,34 @@
+import { render, screen } from "@testing-library/react";
+import { Button } from "../Button";
+
+describe("Button", () => {
+  it("should render same text passed into text prop", async () => {
+    render(<Button text="Button Text" />);
+    const buttonElement = screen.getByText(/Button Text/i);
+    expect(buttonElement).toBeInTheDocument();
+  });
+
+  it("should render Button by Role", async () => {
```

Comment:
> async ()

If these functions don't use promises, there's no need for declaring the function as async.